### ISSUE TITLE
ERROR user rbrtbrnschn is not a collaborator for @codingthrone/codingthrone-bot

### ISSUE THREAD

**Issue by rbrtbrnschn** [release-it/release-it#742]
![image](https://user-images.githubusercontent.com/56562982/108193436-b895a980-7115-11eb-959a-ae73126849fd.png)

I am part of that team, how is it that it still says I am no collaborator?

config:
```
"npm": {
  "publish:" "false"
}
```

**Comment by webpro** [comment]
This is GitHub, right? Do you have more output or logs? Can you please run a debug run, e.g.:

```
DEBUG=release-it:* release-it [...]
```

**Comment by drptbl** [comment]
Hey @webpro,
I'm experiencing same issue on GitHub Actions.

Here is the failing workflow:
https://github.com/Synthetixio/synpress/runs/2155353836?check_suite_focus=true

Workflow file:
https://github.com/Synthetixio/synpress/blob/dev/.github/workflows/release.yml

Proper `NPM_TOKEN` with CI access is present as secret of repository.

Here is the result of a job with `DEBUG` (in case logs from failing workflow above are cleaned up):
https://gist.github.com/drptbl/4e4bb78300d47d874a9ebcff55dd8377

The root cause is this:
```
npm ERR! code E403
npm ERR! 403 403 Forbidden - GET https://registry.npmjs.org/-/package/%40synthetixio%2Fsynpress/collaborators?format=cli - You may not perform that action with these credentials.
npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy.
```

I'm not really sure how this GET call is done, but it seems that `.npmrc` file is not enough for authorization?
I'm able to see the result of this URL in my browser after authenticating on npm.org with my account, so it's proper url and it seems to be missing authorization inside workflow.

**Comment by webpro** [comment]
I just released [v14.5.0](https://github.com/release-it/release-it/releases/tag/14.5.0), which does not bail out on such an error, but prints a warning instead. Please let me know how this works for you.

**Comment by drptbl** [comment]
> I just released [v14.5.0](https://github.com/release-it/release-it/releases/tag/14.5.0), which does not bail out on such an error, but prints a warning instead. Please let me know how this works for you.

Indeed, it fixed the issue. Everything works perfectly now, thanks a lot for crazy fast fix 🚀.

Have a good weekend!

**Comment by webpro** [comment]
Thanks for the feedback, @drptbl! Have a good one.
